src/features/projectsV2/components/microComponents/ProjectInfoSection.tsx

```diff
@@ -0,0 +1,103 @@
+import { useContext, useMemo } from 'react';
+import { useTranslations, useLocale } from 'next-intl';
+import { CountryCode } from '@planet-sdk/common';
+import { useUserProps } from '../../../common/Layout/UserPropsContext';
+import { useTenant } from '../../../common/Layout/TenantContext';
+import { ParamsContext } from '../../../common/Layout/QueryParamsContext';
+import { ProjectInfoProps } from '../ProjectSnippet';
+import { getDonationUrl } from '../../../../utils/getDonationUrl';
+import getFormatedCurrency from '../../../../utils/countryCurrency/getFormattedCurrency';
+import WebappButton from '../../../common/WebappButton';
+import style from '../../styles/ProjectSnippet.module.scss';
+
+const ProjectInfoSection = (props: ProjectInfoProps) => {
+  const {
+    slug,
+    isApproved,
+    isTopProject,
+    countPlanted,
+    purpose,
+    unitType,
+    unitCost,
+    allowDonations,
+    country,
+    currency,
+    unitsContributed,
+  } = props;
+
+  const tCommon = useTranslations('Common');
+  const tCountry = useTranslations('Country');
+  const tAllProjects = useTranslations('AllProjects');
+  const { tenantConfig } = useTenant();
+  const { token } = useUserProps();
+  const locale = useLocale();
+  const { embed, callbackUrl } = useContext(ParamsContext);
+
+  const donateLink = getDonationUrl(
+    tenantConfig.id,
+    slug,
+    token,
+    embed || undefined,
+    callbackUrl || undefined
+  );
+  const donationLabel = useMemo(() => {
+    if (unitType === 'tree' && purpose === 'trees') {
+      return tAllProjects('treeDonated', {
+        count: countPlanted,
+      });
+    } else if (unitType === 'm2' && purpose === 'trees') {
+      return tAllProjects('areaRestored', {
+        area: unitsContributed,
+      });
+    } else {
+      return tAllProjects('areaConserved', {
+        area: unitsContributed,
+      });
+    }
+  }, [countPlanted, unitType, purpose, unitsContributed]);
```

Comment:
Avoid using `countPlanted` - it will be removed in the future, and we'll use `unitsContributed.tree` instead.